image/image.go

```diff
@@ -373,7 +373,9 @@ func bootstrapToRootDir(tsto *ToolingStore, model *asserts.Model, opts *Options,
 		// kernel/os are required for booting
 		if typ == snap.TypeKernel || typ == snap.TypeOS {
 			dst := filepath.Join(dirs.SnapBlobDir, filepath.Base(fn))
-			if err := osutil.CopyFile(fn, dst, 0); err != nil {
+			seedSnaps := "../seed/snaps"
```

Comment:
Nitpick: I would feel better if you had used `path/filepath.Rel` to construct it and otherwise had used variables from `github.com/snapcore/snapd/dirs`

Reply:
I like this, I added code to use this.